### ISSUE TITLE
Conjur OSS install on OpenShift

### ISSUE THREAD

**Issue by diverdane** [cyberark/conjur-authn-k8s-client#248]
## Is your feature request related to a problem? Please describe.

This issue incrementally adds support for OpenShift into the app deployment scripts that were initially created with Issue #239.

**Conjur OSS install on OpenShift**

   The current app deploy scripts use the Conjur OSS Helm chart to install Conjur on a local KinD cluster.
   For this issue, the current scripts will need to be modified to optionally install on OpenShift rather than
   on KinD.

   This may take a bit of effort because the app deploy scripts use the scripts that are in the
   [Conjur OSS on KinD demo scripts](https://github.com/cyberark/conjur-oss-helm-chart/tree/main/examples/kubernetes-in-docker),
   rather than using the Conjur OSS Helm chart directly.

   We may even want to consider separating the Conjur install part of the script from the app deploy script,
   maybe move the existing code that does the Conjur OSS install on KinD, and the new code that
   will do Conjur OSS install on OpenShift into their own separate directories/subdirectories.
   In other words, the installation of Conjur should ideally considered a separate concern
   from the task of deploying applications that use Conjur.

## Describe the solution you would like

See above

## DoD
- [ ] Conjur OSS deploy scripts work on OpenShift platform
